frontend/src/layout/header/Logo.tsx

```diff
@@ -63,7 +69,7 @@ export const Logo: React.FC = () => {
             <img
                 width={small.resolution[0]}
                 height={small.resolution[1]}
```

Comment:
The resolution between dark and light logo might be different, so the correct one should be used here.